package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "hyperswitch",
-  "version": "0.8.8",
+  "version": "0.9.0",
```

Comment:
It does significantly change the logging volume for users that are using "info" in their configs, rather than the "warn" default.